yarn-project/bootstrap.sh

```diff
@@ -171,6 +171,7 @@ function bench_cmds {
   echo "$hash BENCH_OUTPUT=bench-out/tx.bench.json yarn-project/scripts/run_test.sh stdlib/src/tx/tx_bench.test.ts"
   echo "$hash:ISOLATE=1:CPUS=10:MEM=16g:LOG_LEVEL=silent BENCH_OUTPUT=bench-out/proving_broker.bench.json yarn-project/scripts/run_test.sh prover-client/src/test/proving_broker_testbench.test.ts"
   echo "$hash:ISOLATE=1:CPUS=16:MEM=16g BENCH_OUTPUT=bench-out/avm_bulk_test.bench.json yarn-project/scripts/run_test.sh bb-prover/src/avm_proving_tests/avm_bulk.test.ts"
+  echo "$hash:ISOLATE=1:CPUS=16:MEM=16g BENCH_OUTPUT=bench-out/avm_proven_token_test.bench.json yarn-project/scripts/run_test.sh bb-prover/src/avm_proving_tests/avm_proven_token.test.ts"
```

Comment:
If it's fast enough w/check circuit, it might be nice to have this (or the AMM) done in full as tests (all the txs).

Reply:
I'm still not entirely convinced to have this as a benchmark (but also not entirely against): if we want sth "standard" then we may need/want the whole set of txs from constructor etc (without the balance getters). In that way we know how long it takes to prove "a stack of txs to do a token transfer". Otherwise, if we just want to prove opcodes, we have the bulk. The transfer alone I feel doesn't add much.